src/components/form/TextInput/TextInputField.tsx

```diff
@@ -30,8 +29,14 @@ export const TextInputField = memo(
       const { values, errors, touched, handleBlur, setFieldValue } = formikProps
 
       return (
-        <StyledTextInput
-          $displayErrorText={displayErrorText}
+        <TextInput
+          sx={
+            !displayErrorText
+              ? {
+                  '.MuiTextField-root': { marginBottom: '0 !important' },
```

Comment:
Just a guess here, if the elements use a gap between elements rather than a margin bottom, we could get rid of this `sx` definition correct?

Reply:
I guess too! But changing it everywhere in this PR would have been too much imo

Reply:
Ok! But may be worth to check, I have the feeling this rule is obsolete as the TextInput wrapper already apply those rules `flex flex-col gap-1`